requirements/docs.txt

```diff
@@ -3,4 +3,4 @@ sphinxcontrib-httpdomain==1.1.9
 docutils==0.11
 Pygments==1.6
 Sphinx==1.2.2
-mdn-sphinx-theme>=0.3
+sphinx_rtd_theme==0.1.8
```

Comment:
Yeah, the tests blew up with that version. I need to learn what the process is dealing with python packages is - presumably it's not on pyrepo?